go.mod

```diff
@@ -1,8 +1,28 @@
 module github.com/digitalocean/go-libvirt
 
-go 1.15
+go 1.21
 
 require (
-	github.com/stretchr/testify v1.8.1
-	golang.org/x/tools v0.1.12
+	github.com/stretchr/testify v1.6.1
+	github.com/xlab/c-for-go v0.0.0-00010101000000-000000000000
```

Comment:
This won't actually work without the replace directive, and that matters because anyone importing this project into their own project isn't going to get the replace directive from this file. In other words, this project will be broken in any package that imports it.

For this to work, you need to switch the dependency over to point at the forked project, and drop the `replace` directive. You could also vendor the forked c-for-go package, but that only makes sense to do when you want to avoid having a public forked package in our repos.

Reply:
Thanks for the feedback; that's unfortunate. I was hoping to avoid changing the module (and all the of the module imports) in `c-for-go`, but I'll go ahead and use that approach for now. Then when I'm ready to contribute the `c-for-go` changes I'll just do so on a rebased branch.